# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../slmodem/no-install-drivers.patch
# Copyright (C) 2004 - 2006 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- ./Makefile.vanilla	2006-05-22 23:01:03.886987000 +0200
+++ ./Makefile	2006-05-22 23:01:25.828358250 +0200
@@ -25,12 +25,6 @@
 
 install: all
 
-ifndef SUPPORT_ALSA
-all: drivers
-install: install-drivers
-uninstall: uninstall-drivers
-endif
-
 install:
 	$(INSTALL) -D -m 755 modem/slmodemd ${DESTDIR}/usr/sbin/slmodemd
 	$(RM) -rf ${DESTDIR}/var/lib/slmodem
--- slmodem-2.9.11-20051101/drivers/Makefile.vanilla	2006-05-22 20:58:34.899704000 +0200
+++ slmodem-2.9.11-20051101/drivers/Makefile	2006-05-22 20:59:07.029712000 +0200
@@ -31,8 +31,8 @@
 all install uninstall: kernel-ver
 	$(MAKE) $@ KERNEL_VER=$(shell ./kernel-ver)
 
-install: install-devices
-uninstall: remove-devices
+install:
+uninstall:
 
 kernel-ver::
 	$(CC) -I$(KERNEL_DIR)/include -o $@ $@.c
